src/test/regress/expected/multi_mx_router_planner.out

```diff
@@ -864,8 +864,8 @@ SELECT * FROM (
 	(SELECT * FROM articles_hash_mx WHERE author_id = 1)
 	UNION
 	(SELECT * FROM articles_hash_mx WHERE author_id = 2)) uu;
-ERROR:  cannot perform distributed planning on this query
-DETAIL:  Subqueries without group by clause are not supported yet
+ERROR:  cannot push down this subquery
+DETAIL:  Currently all leaf queries need to have same filters on partition column
```

Comment:
I though that we ditched this kind of leaf query checks. Is this specific to union queries?